### PR TITLE
update readme from Adam's patch

### PR DESCRIPTION
**Pull request by HandyKurniawan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,60 @@
...
+
+mct = MCTVChain(5)
+circ = mct.generate_circuit()
+```
+
+With `generate_circuit`, you will get the MCT based on the implementation that you choose.
+
+See [Jupyter Notebook file](https://github.com/QuCoNot/QuCoNot/blob/main/quconot.ipynb) for more in detail examples.
+
+## Authors
```
Comment (adamglos92) [inline README.md:34]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -2,7 +2,60 @@
...
+<sup>2</sup> Facultad de Inform ́atica, Universidad Complutense de Madrid, Spain  
+<sup>3</sup> Rashtreeya Vidyalaya Preuniversity College, India  
+<sup>4</sup> Institute of Theoretical and Applied Informatics, Polish Academy of Sciences, Poland  
+<sup>5</sup> QWorld Association, Tallinn, Estonia  
+<sup>6</sup> Algorithmiq Ltd, Kanavakatu 3C 00160 Helsinki, Finland  
+
+**Corresponding Author:** Adam Glos
+
+
+QuCoNot is an active project, and [other people](https://github.com/qclib/qclib/graphs/contributors) have contributed.
```
Comment (adamglos92) [inline README.md:55]: We should not mention these people there. We merely used their implementation
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -2,7 +2,60 @@
...
+
+If you are doing research using QuConot, please cite our project.
+We use a [CITATION.cff](https://citation-file-format.github.io/) file, so you can easily copy the citation information from the repository landing page.
```
Comment (adamglos92) [inline README.md:58]: Instead, we should have link to the paper, not to the repo.

**Review by adamglos92 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (HandyKurniawan) [reply README.md:55]: i forgot to change the link name actually i dont want to mention them

File: README.md [hunk 4]
```diff
@@ -2,7 +2,31 @@
...
+You just need to import it, and you can start use it
+
+```
+from quconot.implementations import MCTVChainDirty
+
+mct = MCTVChain(5)
+circ = mct.generate_circuit()
+```
+
+With `generate_circuit`, you will get the MCT based on the implementation that you choose.
```
[the remark this answers is not in the document]
Comment (HandyKurniawan) [reply README.md:32]: done

File: README.md [hunk 5]
```diff
@@ -2,7 +2,86 @@
...
+    verify_circuit_strict_dirty_wasting_separable
+)
+
+tested_matrix = usim.run(circ).result().get_unitary().data
+ref_unitary = get_ref_unitary(control_no)
+verify_circuit_strict_dirty_non_wasting(tested_matrix, ref_unitary)
+verify_circuit_strict_clean_non_wasting(tested_matrix, ref_unitary)
+```
+
+Note that, we need to update the circuit into unitary matrix `tested_matrix = usim.run(circ).result().get_unitary().data` .
```
Comment (adamglos92) [inline README.md:58]:
```suggestion
Note that verification functions requires the unitary to be in a form of numpy array, not `QuantumCircuit`:  `tested_matrix = usim.run(circ).result().get_unitary().data` .
```

**Review by adamglos92 — APPROVED** [review]

**Pull request merged**